aten/src/TH/vector/VSX.cpp

```diff
@@ -2450,6 +2461,7 @@ void test_THFloatVector_divs_VSX()
     free(y_standard);
     free(y_optimized);
     free(x);
+    free(y);
```

Comment:
is y defined and allocated in this function?

Reply:
Thanks for the review. Corrected.

Reply:
I'm surprised as to how it compiled.